test/acceptance/tests.py

```diff
@@ -228,17 +227,38 @@ def do_staff_assessment(self, number_to_assess=0, options_selected=OPTIONS_SELEC
                 will grade all available submissions.
         """
         self.staff_area_page.visit()
-        self.staff_area_page.expand_staff_grading_section()
+        self.staff_area_page.click_staff_toolbar_button("staff-grading")
+        # Get the counts before checking out a submission for assessment.
         start_numbers = self.staff_area_page.available_checked_out_numbers
+        # Check out a submission.
+        self.staff_area_page.expand_staff_grading_section()
+        # Checked out number should increase, ungraded decrease.
```

Comment:
As part of my last PR, the ungraded/checked out numbers update when you check out an assessment for grading.